src/Lexer.ts

```diff
@@ -0,0 +1,98 @@
+// @author Anush kumar N <anush.kumar@codingmart.com>
+
+import TokenType from './Token';
+
+type arrayOfString = Array<string>;
```

Comment:
Why? Can't the call sites just directly reference Array<string>?

Reply:
this error: `Generic type 'Array<T>' requires 1 type argument(s).` will pop if we don't declare what are the contents of the array.

Reply:
ah i know. What I meant was why alias Array<string>? can't you just use `Array<string>` throughout the code? Many programming languages have similar constructs so this will look familiar to most programmers, but arrayOfString won't.

Reply:
Yes got your point, i have updated it already.